corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/form.less

```diff
@@ -138,6 +138,12 @@
       float: left;
       margin-right: 10px;
     }
+    span,.caption {
```

Comment:
@AddisonDunn I did mean additive to make it more specific. But upon reflection I think `.webapp-markdown-output` is more appropriate since I am targeting markdown output.

Reply:
@AddisonDunn it is limited to group headers. There will be another story for an appearance attribute for the rest.
The additions in this PR are nested in `.gr-header`.
```
.gr-header {
    .collapsible-icon-container {
      float: left;
      margin-right: 10px;
    }
    .webapp-markdown-output img {
      height: 1em;
      vertical-align: baseline;
    }
  }
  ```

Reply:
I see it's no longer relevant but AFAIK `span,.caption` is equivalent to `span, .caption` selecting both `<span>` and `class="caption"` elements. `span.caption` would be additive to select `<span class="caption">` elements.